dask/array/tests/test_rechunk.py

```diff
@@ -164,3 +164,9 @@ def test_rechunk_with_empty_input():
     x = da.ones((24, 24), chunks=(4, 8))
     assert x.rechunk(chunks={}).chunks == x.chunks
     assert raises(ValueError, lambda: x.rechunk(chunks=()))
+
+
+def test_rechunk_with_null_dimensions():
+    x = da.ones((24, 24), chunks=(4, 8))
+    assert (x.rechunk(chunks=(None, 4)).chunks ==
+            da.ones((24, 24), chunks=(24, 4)).chunks)
```

Comment:
Hrm, I would have interpreted this as "don't change the current chunking of the first dimension"